### ISSUE TITLE
Generated Localizations file is incomplete if input file contains certain characters

### ISSUE THREAD

**Issue by andriichernenko** [JiriTrecak/Laurine#47]
Spent some time today figuring out why out of 200+ strings Laurine generated localizations only for 17. Turned out it was caused by an illegal character at the end of one line which looks like normal space (I asked a colleague with Windows machine to translate the file for me):

```swift
let bad = " "
let good = " "

bad == good // false

print(bad.utf8.map { String($0, radix: 16) })  // ["c2", "a0"]
print(good.utf8.map { String($0, radix: 16) }) // ["20"]
```

I have no idea what this thing is, but it breaks `NSDictionary(contentsOfFile:)` on which Laurine relies to read the input file. 

If the character is in the middle of string the result is `nil`, if it's in the end `NSDictionary` quietly fails to read further and returns what was read so far.

I used the latest version from `master`.

**Comment by JiriTrecak** [comment]
Thank you for your feedback @deville, I will try to investigate. Most probably I will have to go through entire file before parsing it out and just replace those characters, but I'll have to try to figure out some generic way (and probably also give some error message it all else fails.

Have a great day,
Jiri

**Comment by JiriTrecak** [comment]
This will be fixed in upcoming Laurine 2.0, which will have a debug option that will look for duplicities and errors in the file, and also probably strict parsing mode that will fail when something like this is encountered. Please refer to #52 for any further comments.
